tools/internal_ci/linux/grpc_xds_k8s_lb.sh

```diff
@@ -101,6 +101,7 @@ run_test() {
     --secondary_kube_context="${SECONDARY_KUBE_CONTEXT}" \
     --server_image="${SERVER_IMAGE_NAME}:${GIT_COMMIT}" \
     --client_image="${CLIENT_IMAGE_NAME}:${GIT_COMMIT}" \
+    --testing_version=$(echo "$KOKORO_JOB_NAME" | sed -E 's|^grpc/core/([^/]+)/.*|\1|') \
```

Comment:
Please add a comment explaining what's going on here.

Reply:
Comment added to all 7 Kokoro script. It explains how the version is fetched, and why we need it.